js/index.js

```diff
@@ -0,0 +1,29 @@
+let body = document.querySelector('body');
+let mobileNavMenu = document.querySelector('.mobile-nav');
+
+function openMenu() {
+  mobileNavMenu = document.querySelector('.mobile-nav');
+  body = document.querySelector('body');
+  mobileNavMenu.style.display = 'block';
+  body.style.overflow = 'hidden';
+}
+
+function closeMenu() {
+  mobileNavMenu = document.querySelector('.mobile-nav');
+  body = document.querySelector('body');
+  mobileNavMenu.style.display = 'none';
+  body.style.overflow = '';
+  console.log(mobileNavMenu.style.display);
```

Comment:
- [ ] Please don't forget to remove console.log() from your code before pushing it to your repository. it reduces the performance and it can make confidential data visible to anyone using the browser tools to look at your website.